manim/config/logger.py

```diff
@@ -49,51 +64,55 @@ def parse_theme(fp):
         printf(
             "[logging.level.error]It seems your colour configuration couldn't be parsed. Loading the default color configuration...[/logging.level.error]"
         )
-    return customTheme, theme
-
-
-config_items = _run_config()
-config_parser, successfully_read_files = config_items[1], config_items[-1]
-try:
-    customTheme, themedict = parse_theme(successfully_read_files)
-    console = Console(
-        theme=customTheme,
-        record=True,
-        height=themedict["log.height"],
-        width=themedict["log.width"],
-    )
-except KeyError:
-    console = Console(record=True)
-    printf(
-        "[logging.level.warning]No cfg file found, creating one in "
-        + successfully_read_files[0]
-        + " [/logging.level.warning]"
+    return customTheme
+
+
+def set_rich_logger(config_logger):
+    """Will set the RichHandler of the logger.
```

Comment:
Hmm I _think_ it's because logger.py location has been changed by a recent config refactor. Any thought about this @leotrs ?

Reply:
No, it's because sphinx doesn't like that I renamed the config module during import as `import manim.config as _config`. I'm working on a fix that will change that (but I have 0 time this week so).

Let's merge this as is, or just remove the logger from documentation for now.

Reply:
+1, it was not meant as a blocking comment.